### ISSUE TITLE
Location Index

### ISSUE THREAD

**Issue by astro71** [CadeBrown/PokemonGoBot-Manager#1]
when i type the number in the index 
![screen shot 2016-08-02 at 12 04 07 pm](https://cloud.githubusercontent.com/assets/20786466/17335777/3d90019c-58a9-11e6-8354-2ac41ba393ae.png)

**Comment by cadebrown** [comment]
Can you paste your locations.json here?

**Comment by astro71** [comment]
{
  "0. Santa Monica Pier - Great General Farming": {
    "index": "0",
    "name": "Santa Monica Pier",
    "info": "Great for water types, electabuzz, pokestops, overall great for general grinding",
    "loc": "34.008344620842834,-118.49794507026672"
  },
  "1. Good farming spot": {
    "index": "2",
    "name": "Sydney",
    "info": "General farming",
    "loc": "-33.866609,151.209120"
  },
  "2. Central Park - Snorlax Spot": {
    "index": "3",
    "name": "Central Park SW",
    "info": "Snorlax spawned here",
    "loc": "40.76976805976344,-73.979771733284"
  },
  "3. London": {
    "index": "4",
    "name": "Densest area of london",
    "info": "Good for XP/Snorlax",
    "loc": "51.495011293466746,-0.13200759887695312"
  }
}

**Comment by astro71** [comment]
could it be my google key? just starting with coding, don't really know much at all haha

**Comment by cadebrown** [comment]
If you don't have one, just google how to get one, and paste that in configs/config.json where it says

**Comment by astro71** [comment]
is this correct?
![screen shot 2016-08-02 at 12 16 49 pm](https://cloud.githubusercontent.com/assets/20786466/17336195/0cf1f480-58ab-11e6-88fd-1543b2e1aa11.png)

**Comment by cadebrown** [comment]
Yes. So is it just not starting at all? Did you change your bots.json?

**Comment by cadebrown** [comment]
Also, are you on the newest commit?

**Comment by astro71** [comment]
have a feeling the bots.json might be the problem, should the one with "~" have bot info filled in aswell?
![screen shot 2016-08-02 at 12 20 27 pm](https://cloud.githubusercontent.com/assets/20786466/17336305/929b0a40-58ab-11e6-91d3-cdb82dc3b1c2.png)

**Comment by astro71** [comment]
really precaite the help, newbie here

**Comment by cadebrown** [comment]
Have you put your email and password in the json?

**Comment by astro71** [comment]
yes, bots.json has two accounts email and pass filled out (bots.json~ does not however).

only other thing i can think of is i only have a google key for one of the accounts in config as there was only a spot for one

**Comment by cadebrown** [comment]
Try and reclone the newest version in a seperate folder. These few updates last night fixed a lot of bugs.

**Comment by astro71** [comment]
![screen shot 2016-08-02 at 12 54 42 pm](https://cloud.githubusercontent.com/assets/20786466/17337382/5fa70418-58b0-11e6-9921-de3df09ec3ef.png)

**Comment by cadebrown** [comment]
Follow the the main guide on the front of this repo. You have to copy some files (remove the .examples from bots, locations, and your config, and then change the values).

**Comment by astro71** [comment]
Will do, was running dev before. setting up main now :)

**Comment by astro71** [comment]
![screen shot 2016-08-02 at 2 16 55 pm](https://cloud.githubusercontent.com/assets/20786466/17339877/d7cd10d0-58bb-11e6-8522-d09b8f2750db.png)
